embuilder.py

```diff
@@ -116,6 +117,14 @@ def build(src, result_libs, args=[]):
       build('''
         int main() {}
       ''', [os.path.join('ports-builds', 'zlib', 'libz.a')], ['-s', 'USE_ZLIB=1'])
+    elif what == 'libpng':
+      build('''
+        int main() {}
+      ''', [os.path.join('ports-builds', 'libpng', 'libpng.bc')], ['-s', 'USE_ZLIB=1', '-s', 'USE_LIBPNG=1'])
```

Comment:
i see. yes, it would be nice to add deps to ports. we'd need to think about that.